pkg/api/v1/conversion.go

```diff
@@ -670,3 +630,53 @@ func Convert_v1_ResourceList_To_api_ResourceList(in *ResourceList, out *api.Reso
 	}
 	return nil
 }
+
+func AddFieldLabelConversionsForEvent(scheme *runtime.Scheme) error {
+	return api.Scheme.AddFieldLabelConversionFunc("v1", "Event",
```

Comment:
All of these need to be using `scheme`-- you're not actually putting them in the federation scheme right now!